controllers/fenceagentsremediation_controller.go

```diff
@@ -50,6 +50,7 @@ func (r *FenceAgentsRemediationReconciler) SetupWithManager(mgr ctrl.Manager) er
 
 //+kubebuilder:rbac:groups=core,resources=pods/exec,verbs=create
 //+kubebuilder:rbac:groups=core,resources=pods,verbs=get;list;watch;update;delete;deletecollection
+//+kubebuilder:rbac:groups="",resources=nodes,verbs=list;watch
```

Comment:
I tried with and without `get` and saw that it is working, thus I though of adding less verbs for rbac.

Reply:
> I though of adding less verbs for rbac

And in general that's a good approach!
But a `list` without `get` doesn't make sense, you have access to all resources already.
Thinking more about it: the other way around it can make sense though: a `get` without `list` / `watch`. That's a difference IMHO. Even more when you also restrict the `get` with a resource name (e.g. you are only allowed to read a secret with a specific name)

Reply:
Having the three verbs is working, so I tried to minimize it:

1. `get;watch` failed and couldn't move on in the reconcile (to run the FA)
`W0511 11:22:40.347954 1 reflector.go:324] sigs.k8s.io/controller-runtime/pkg/cache/internal/informers_map.go:250: failed to list *v1.Node: nodes is forbidden: User "system:serviceaccount:openshift-operators:fence-agents-remediation-controller-manager" cannot list resource "nodes" in API group "" at the cluster scope
20`
`E0511 11:22:40.347984 1 reflector.go:138] sigs.k8s.io/controller-runtime/pkg/cache/internal/informers_map.go:250: Failed to watch *v1.Node: failed to list *v1.Node: nodes is forbidden: User "system:serviceaccount:openshift-operators:fence-agents-remediation-controller-manager" cannot list resource "nodes" in API group "" at the cluster scope`

2. `get;list` failed but I could move on in the reconcile (to run the FA) 
`E0511 11:44:07.909025 1 reflector.go:138] sigs.k8s.io/controller-runtime/pkg/cache/internal/informers_map.go:250: Failed to watch *v1.Node: unknown (get nodes)`

To conclude, I tend to just stick with `get;list;watch` verbs.